cmd/neutron_query_relayer/cmd/root.go

```diff
@@ -0,0 +1,26 @@
+package cmd
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+// rootCmd represents the base command when called without any subcommands
+var rootCmd = &cobra.Command{
+	Use:   "neutron_query_relayer",
+	Short: "Neutron Interchain Query Relayer",
```

Comment:
Neutron Interchain Quer**ies** Relayer

Reply:
Why? it's build as a binary named `neutron_query_relayer`

Reply:
If it requires a lot of changes, it's better to do it later of course